### PR TITLE
Completed Part2

### PR DESCRIPTION
**Pull request by JordanWav**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -505,6 +746,35 @@ struct Garage
...
+    if ( myCar.carType == "Ford" )
+    {
+        return true;
+    }
+
+    else
+    {
+        return false;
+    }
+}
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 2]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
-    5) shelves
-3 things it can do:
-    1) open garage door
-    2) use workbench
-    3) park car in garage
- */
+    else
+    {
+        return false;
+    }
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 3]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+    if ( carA.ownerOfCar == "Me" || carA.ownerOfCar == "Jason")
+    {
+        return true;
+    }
+
+    else
+    {
+        return false;
+    }
+}
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 4]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+    if ( carB.numWheels > 1 )
+    {
+        return true;
+    }
+
+    else
+    {
+        return false;
+    }
+}
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 5]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+    if ( emptyBoxA.weightOfBox < 1.f )
+    {
+        emptyBoxA.numBoxes += 1;
+        return emptyBoxA.numBoxes;
+    }
+
+    else
+    {
+        return 0;
+    }
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 6]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+{
+    if ( equipment.equipmentType != "pullup bar")
+    {
+        return true;
+    }
+    
+    else
+    {
+        return false;
+    }
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 7]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+{
+    if ( benchpress.workoutType == "strength training")
+    {
+        return 20.f;
+    }
+
+    else
+    {
+        return 0.f;
+    }
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 8]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+{
+    if ( dumbbell.numDumbbells > 1 )
+    {
+        return 2;
+    }
 
+    else
+    {
+        return 0;
+    }
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 9]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+{
+    if ( myWorkbench.numBrokenTools > 0 )
+    {
+        return true;
+    }
+
+    else
+    {
+        return false;
+    }
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 10]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+{
+    if ( myWorkbench.numTools > 0 )
+    {
+        return true;
+    }
+
+    else
+    {
+        return false;
+    }
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 11]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+    if ( myMonitor.numPossibleInputs >= 2)
+    {
+        return true;
+    }
+
+    else
+    {
+        std::cout << "no HDMI input available" << std::endl;
+        return false;
+    }
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 12]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+    if ( myMonitor.numUserProfiles == 2 )
+    {
+        return "Profile 2";
+    }
+
+    else
+    {
+        return "Profile 1";
+    }
+}
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 13]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+    {
+        std::cout << "Please Create a New Track" << std::endl;
+        return 2;
+    }
+
+    else
+    {
+        std::cout << "Something Went Wrong... Please Try Again" << std::endl;
+        return 3;
+    }
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 14]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
-    3) switch to HDMI input
- */
+std::string LogicPro::createInstrumentTrack(LogicPro trackA)
+{
+    if ( trackA.numTracks < 1000 )
+    {
+        return "new track";
+    }
+
+    else
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 15]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+    if ( myXbox.amountOfSpace >= haloInfinite.sizeOfGame )
+    {
+        return true;
+    }
+
+    else
+    {
+        return false;
+    }
+}
```
Comment (matkatmusic) [inline main.cpp]: `Xbox::downloadGame` is a member function. 
you do not need to declare another instance of `Xbox` inside of this function to use member variables. 
When you call this function with an instance of `Xbox`, you can use the caller's instance's member variables.
i.e.
```
Xbox xbox;
xbox.downloadGame( Xbox::Game() );
//the 'amountOfSpace' used inside the 'downloadGame' function is the one inside 'xbox'
```
- [x] revise

File: main.cpp [hunk 16]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+    if ( skyrim.gameCategory == "RPG" )
+    {
+        return true;
+    }
+
+    else
+    {
+        return false;
+    }
+}
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 17]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+        numPoints += 5;
+        return true;
+    }
+
+    else
+    {
+        numPoints += 0;
+        return false;
+    }
+}
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 18]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+    if ( gameName == "Valid" )
+    {
+        return true;
+    }
+
+    else
+    {
+        return false;
+    }
+}
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 19]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+    {
+        return true; 
+    }
+
+    else
+    {
+        return false;
+    }
+
+}
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 20]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+
+void House::openFrontDoor(std::string door)
+{
+    if ( door == "front door" )
+    {
+        std::cout << "Need to open front door" << std::endl;
+    }
+
+    else
+    {
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 21]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+    if( myHouse.numBeds == 0 )
+    {
+        return true; // furnish the house
+    }
+
+    else
+    {
+        return false; // don't furnish the house
+    }
+}
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 22]
```diff
@@ -125,372 +170,568 @@ struct CarWash
...
+    
+    if( microwaveTime >= 45.f )
+    {
+        ready = true;
+    }
+
+    else
+    {
+        ready = false;
+    }
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

File: main.cpp [hunk 23]
```diff
@@ -108,6 +108,51 @@ struct CarWash
...
+{
+    howFast = 10;
+    
+    if( startWithLeftFoot == true )
+    {
+        leftFoot.stepForward();
+        rightFoot.stepForward();
+    }
+
+    else
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 15 above)
Comment (JordanWav) [reply main.cpp]: I don't completely understand what you mean. I made changes to this one but I don't know if I did it correctly.

**Review by matkatmusic — CHANGES_REQUESTED** [review]
Don't forget to push your changes.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**